packages/shared/ReactComponentStackFrame.js

```diff
@@ -33,17 +34,26 @@ export function describeBuiltInComponentFrame(name: string): string {
       } catch (x) {
         const match = x.stack.trim().match(/\n( *(at )?)/);
         prefix = (match && match[1]) || '';
+        suffix =
+          x.stack.indexOf('\n    at') > -1
+            ? // V8
+              ' (<anonymous>)'
+            : // JSC/Spidermonkey
+            x.stack.indexOf('@') > -1
+            ? '@unknown:0:0'
```

Comment:
This can really be anything include empty `'@'` but then Firefox can still present it as `:0` which is maybe confusing. So the string `unknown` is just used here to communicate that we don't know the location.